### DISCUSSION TITLE
Collection of POIs & TRACKS from users for users (Sammlung von POIs & TRACKS von Benutzern für Benutzer)

### DISCUSSION THREAD

**Discussion by DIrgi** [devemux86/cruiser#488] [Show and tell]
**Preface:**
Since I collected in the past years some useful POIs & tracks which were of interest for me and maybe others, I'd like to share them with you. This tread can be used for everyone to share his POIs & tracks with others as well. A kind of a maybe interesting and useful POI & track collection so to say . A small description and maybe a picture will be fine. Have fun.
<br>
**Vorwort:**
Nachdem ich im Laufe der Jahre einige für mich und vielleicht auch für andere nützliche POIs & Tracks gesammelt habe, möchte ich diese gerne zur Verfügung stellen. Dieser Tread soll hier so angelegt sein, das Jeder der möchte hier ebenso seine POIs & Tracks für Andere ablegen kann. Eine Art von interessanter und vielleicht nützlicher POI & Track Sammlung sozusagen. Mit einer kleinen Beschreibung um was es geht und vielleicht ein Bild. Das wäre schön. Viel Spaß.
<br><hr>
### Instructions for POIs (Anleitung für POIs)
(tracks see below)

**Requirements:**
For a comfortable and easy use, I recommend to offer your POIs as *.gpx file, only. Either direct as gpx file or if packed, please use the most common ZIP format. Further on to avoid issues, please check your POIs if and how the will work/look on the cruiser app, before you upload them. Thank you
<br>
**Vorgaben:**
Um es für alle einheitlich und leicht in der Anwendung zu gestalten, empfehle ich ausschließlich die POI Daten als *.gpx Datei anzubieten. Entweder unverpackt als GPX Datei oder aber gepackt. Es sollte hier der Einfachheit halber nur das gängige ZIP Format verwendet werden. Des weiteren wäre wünschenswert, wenn auch bereits im Vorfeld geprüft wurde ob und wie die POIs auf Cruiser angezeigt werden, bevor die Dateien hochgeladen werden.
Danke

**Install Instruction - How to?**
This should be a general instruction how to install the offered POIs into your Cruiser App. What needs to be taken into account and which settings/changings are useful to have the best outcome at the end. Witt this instruction there should be no more need for uploaders to explain how it works. They simply can share their data write a small description and...that's it.
<br>
**Installationsanleitung: Wie funktioniert es?**
Dies soll eine generelle Anleitung sein, wie man die hier angebotenen POIs in seine Cruiser App installiert, was es dabei zu beachten gibt, und was man für evtl. sinnvolle Einstellungen/Änderungen vornehmen sollte um sie auch angezeigt zu bekommen. Dank dieser Anleitung sollte dann Niemand mehr erklären müssen, wie die Vorgehensweise ist und braucht nur noch seine Daten anzubieten mit einer kleinen Beschreibung.
<br><hr>
**English:**---------------------------------

1. Download the provided gpx or zip file. If it is a ZIP* file you have to extract the file before use. Normally every modern computer can handle this nowadays easily.
_* update: ZIP files can be handled and need no extraction_
	- If you download directly to your android device, make sure you are aware how to handle data there and especially how to extract.
	- If you first save the data on your computer, there are several ways to bring the data to your android device. Either via a common cloud (i.e. Google drive), or simply use a messenger (i.e. WhatsApp, Telegram)to share data with yourself from different devices and so on....
	- Make sure you know where to find the file later on on your android device.
2. Open the Cruiser App, click on the symbol with the 3 bars on the left.
	- Select "my places" and "Favorites"
	- Click now to the right on the "3 dots" (it doesn't matter which if there are more)and select "new group". Enter now any name you like and click "accept" (the hook)
3. Now you see the "new group". Click again on the "3 dots" from the new group.
	- Select now "import". After that you have to know or go to the location where your Android device stored the previous gpx file.
	- Quite often you find something in the download folder, or if you stored it in your "google drive", open it via the cloud. Unfortunately there are many possibilities I cannot cover here.
	- Once you found the file, import it by selecting "update group". If everything works fine and the file has been perfectly prepared for you, the POIs will be now available in the new folder and the folder will have the name of the file now (this you can change)
4. To finalize the process we should do some further changings/settings to have the best outcome.
        - For that reason press once again the "3 dots" of the group. Now you can change here the name if you like.
	- Each imported POI symbols receive automatically a * star symbol. If you want to make it more unique and to differentiate, you can click on "symbol" and switch to one of the displayed symbols there. Find something that fits best to your mind - Unfortunately you cannot have more symbols there which might fit better or even your own - maybe Emux "has an idea for the future ;-)"
	- Once done, you can give the POI symbols as well other colors to differentiate them even more. Click "colors" and select your desired color.
	- Last but not least, the most important one - We would like to have and see them on the map. Therefore click "Show on map" and activate "show on map". same with "use in navigation" (should be self explaining). Finally it's up to you at which zoom level they should appear. However, the base setting display them from an early zoom out level and depending of the amount or POIs they can cover your whole map. To make it more comfortable and let the POIs appear first when you zoomed more in, I would recommend a zoom level of 10 or even higher. However, you have to select which fits best to you.

🙂 **Congratulations**, you are done. Don't forget to click on the safe symbol (bottom right) before leaving otherwise everything is "gone".
Everything mentioned above can be changed at any time again. If you want to have some POIs temporary not visible in your map,
simply just deactivate them (you need not to delete them), by deactivating the "show on map" and/or "show on navigation". That's it, have fun.
<br>

**Deutsch:** ---------------------------------
1. Lade die angebotene gpx ode zip Datei herunter. Wenn es eine ZIP* Datei ist, musst Du sie zunächst entpacken. Normalerweise kann das heute jeder moderne Computer.
_*update: Cruiser kann mit ZIP Dateien umgehen. Sie müssen somit nicht entpackt werden_
	- Wenn Du die Datei direkt auf Dein Android Gerät herunterlädst, vergewissere Dich, das Du weißt wie Du dort damit umgehst und wie Du sie dort auch entpacken kannst.
	- Wenn Du die Datei zunächst auf Deinen Computer speicherst, gibt es mehrere Möglichkeiten, diese im Anschluss auf Dein Android zu bringen. Entweder über eine gemeinsame
	  Cloud (zB Google Drive) oder einfach zB über einen Messenger Dienst als Dateianhang, den Du mit Dir selbst über verschiedene Geräte teilst. Geht zB mit WhatsApp oder Telegram usw.
	- Vergewissere Dich, das Du die Datei dann später auch wieder auf Deinem Android Gerät auffinden wirst.
2.	Öffne die Cruiser App. Klicke auf das linke Symbol mit den "drei Balken".
	- Wähle "Meine Orte" und im Anschluss "Favoriten" aus.
	- Klicke nun zur Rechten auf die "3 Punkte" (egal welche, wenn da mehr sind) und wähle "Neue Gruppe" aus. Tippe nun irgendetwas ein und wähle "Akzeptieren" (das Häkchen)
3.  Jetzt erscheint die "neue Gruppe". Klicke abermals auf die "3 Punkte" der neune Gruppe.
	- Wähle nun "Importieren" aus. Danach musst Du natürlich wissen, wo sich Deine Datei befindet. Oft liegt sie unter downloads, oder wenn sie im Google Drive ist, findest Du
	  sie über die Cloud. Leider gibt es zu viele Möglichkeiten, die ich hier nicht aufzeigen kann.
	- Wenn Du die Datei gefunden hast, importiere sie, indem Du "Gruppe aktualisieren" anwählst. Wenn alles gut läuft und die Datei gut vorbereitet wurde, sind die POIs nun in Deinem
	  Gruppenordner verfügbar und der Ordner hat automatisch den Namen der Datei bekommen (das kannst Du ändern)
4. Um den Prozess abzuschließen, sollten wir noch ein paar Einstellungen ändern/vornehmen, damit wir das beste Ergebnis erzielen.
	- Aus diesem Grund wähle wieder die "3 Punke" der neuen Gruppe an, jetzt kannst Du hier den Namen wechseln, wenn Du möchtest.
	- Jede importierte POI Symbole bekommen zunächst automatisch das * Stern Zeichen verpasst. Wenn Du es besser unterscheiden möchtest, kannst Du auf "Symbol" klicken und Dir ein
	  anderes aus der Liste heraussuchen, was am besten für Dich passt. Leider gibt es nur eine begrenzte Auswahl und auch Symbole, die nicht immer ganz so passen. Auch eigen Symbole
	  kann man nicht verwenden - Vielleicht ändert der Entwickler Emux das irgendwann einmal ab ;-) - Wer weiß?
	- Wenn das erledigt ist, können wir den Symbolen noch eine andere Farbe geben durch anwählen von "Farbe". Das macht sie auf der Karte noch besser unterscheidbar.
	- Zu guter Letzt das wohl wichtigste: Wir möchten die POIs natürlich auch auf der Karte sehen. Dafür wählen wir "Auf der Karte anzeigen" an und aktivieren die 
	  Option "Auf der Karte anzeigen" aus. Ebenso "Verwendung in der Navigation" (sollte alles selbsterklärend sein). Zuletzt stellen wir noch das Zoom Level ein. Das ist die Ebene,
	  aber der beim hineinzoomen die POIs sichtbar werden. Von Haus aus ist das immer aus einem hohen Zoom heraus festgelegt. Bei vielen POIs kann es dann aber passieren, dass man
	  die Karte nicht mehr sieht. Aus diesem Grund empfehle ich hier ein Zoomlevel von 10 oder größer. Einfach mal selbst testen, wie es am besten passt.
	  
🙂 **Glückwunsch**. Du bist fertig. Ganz wichtig: Vergesse nicht zu Speichern indem DU unten rechts das Speichern Symbol anklickst, sonst ist wieder alles weg. Alles was oben beschrieben wurde, kann nachträglich auch wieder geändert werden. Wenn Du manche POIs vorübergehend nicht anzeigen möchtest, brauchst Du sie nicht gleich zu löschen, deaktivieren sie einfach wieder, so wie Du sie zuletzt in der Karte bei der Navigation aktiviert hast. Das war's. Viel Spaß.

<hr>

### Instructions for Tracks (Anleitung für Tracks)
(POIs see above)

**Requirements:**
For a comfortable and easy use, I recommend to offer your Tracks as *.gpx file, only. Either direct as gpx file or if packed, please use the most common ZIP format. Further on to avoid issues, please check your Tracks if and how the will work/look on the cruiser app, before you upload them. Thank you
<br>
**Vorgaben:**
Um es für alle einheitlich und leicht in der Anwendung zu gestalten, empfehle ich ausschließlich die Track Daten als *.gpx Datei anzubieten. Entweder unverpackt als GPX Datei oder aber gepackt. Es sollte hier der Einfachheit halber nur das gängige ZIP Format verwendet werden. Des weiteren wäre wünschenswert, wenn auch bereits im Vorfeld geprüft wurde ob und wie die Tracks auf Cruiser angezeigt werden, bevor die Dateien hochgeladen werden.
Danke

**Install Instruction - How to?**
This should be a general instruction how to install the offered Tracks into your Cruiser App. What needs to be taken into account and which settings/changings are useful to have the best outcome at the end. With this instruction there should be no more need for uploaders to explain how it works. They simply can share their data write a small description and...that's it.
<br>
**Installationsanleitung: Wie funktioniert es?**
Dies soll eine generelle Anleitung sein, wie man die hier angebotenen Tracks in seine Cruiser App installiert, was es dabei zu beachten gibt, und was man für evtl. sinnvolle Einstellungen/Änderungen vornehmen sollte um sie auch angezeigt zu bekommen. Dank dieser Anleitung sollte dann Niemand mehr erklären müssen, wie die Vorgehensweise ist und braucht nur noch seine Daten anzubieten mit einer kleinen Beschreibung.
<br><hr>
**English:**---------------------------------

1. Download the provided gpx or zip file. If it is a ZIP* file you load it into cruiser as well without extraction since cruiser support ZIP files
	- If you download directly to your android device, make sure you are aware how to handle data there and especially how to extract.
	- If you first save the data on your computer, there are several ways to bring the data to your android device. Either via a common cloud (i.e. Google drive), or simply use a messenger (i.e. WhatsApp, Telegram)to share data with yourself from different devices and so on....
	- Make sure you know where to find the file later on on your android device.
2. Open the Cruiser App, click on the symbol with the 3 bars on the left.
	- Select "my places" and "Tracks"
	- Make sure you are in the 3 dots folder (see top left the 3 big dots)
	- If it is the first time you are opening the tracks, and empty folder will be already in there, called tracks.
	- Click now to the right on the "3 dots"  (of the track folder) and select "new group". Enter now any name you like (I recommend road closures) and click "accept" (the hook)
3. Now you see the "new group". Click again on the "3 dots" from the new group.
	- Select now "import". After that you have to know or go to the location where your Android device stored the previous gpx file.
	- Quite often you find something in the download folder, or if you stored it in your "google drive", open it via the cloud. Unfortunately there are many possibilities I cannot cover here.
	- Once you found the file, import it (this might takes a little bit longer then a POI import since Track files are often larger). If everything works fine and the file has been perfectly prepared for you, the Tracks will be now available in the new folder. 
	- In comparison the tracks folder structure slightly differs to the POIs structure. In the POI structure you import one file which might contain many single POIs. In cruiser the folder name will have the gpx import file name where the content of the folder will list each single POI containing in the gpx file you recently imported (this can be a lot). 
	When it comes now to the tracks. The way is similar but as well different. You build a new folder called "road closures" there you import the track gpx file. Now you will see inside the folder this one gpx file only. This means the "road closure" folder act somehow now like a head folder where you can install even more road closure tracks (maybe from other countries). 
	In comparison to the POIs all tracks which are still in the one gpx file remain there while the POI become extracted as singe points.
4. To finalize the process we should do some further changings/settings to have the best outcome.
        - For that reason press once again the "3 dots" of the group.
	- You can give the Tracks a unique color to differentiate them from others. Click "colors" and select your desired color.
	- Last but not least, the most important one - We would like to have and see them on the map. Therefore click "Show on map" (first click on the 3 dots again)and activate "show on map". Finally it's up to you at which zoom level they should appear. However, the base setting display them from an early zoom out level and depending of the amount of Tracks they can cover your whole map. To make it more comfortable and let the Tracks appear first when you zoomed more in, I would recommend a zoom level of 10 or even higher. However, you have to select which fits best to you.

🙂 **Congratulations**, you are done. Don't forget to click on the safe symbol (bottom right) before leaving otherwise everything is "gone".
Everything mentioned above can be changed at any time again. If you want to have some Tracks temporary not visible in your map,
simply just deactivate them (you need not to delete them), by deactivating the "show on map" and/or "show on navigation". That's it, have fun.
<br>

**Deutsch:** ---------------------------------
1. Lade die angebotene gpx ode zip Datei herunter. Cruiser kann mit  ZIP Dateien umgehen, Du musst sie nicht mehr entpacken.
	- Wenn Du die Datei direkt auf Dein Android Gerät herunterlädst, vergewissere Dich, das Du weißt wie Du dort damit umgehst und wie Du sie dort auch entpacken kannst.
	- Wenn Du die Datei zunächst auf Deinen Computer speicherst, gibt es mehrere Möglichkeiten, diese im Anschluss auf Dein Android zu bringen. Entweder über eine gemeinsame
	  Cloud (zB Google Drive) oder einfach zB über einen Messenger Dienst als Dateianhang, den Du mit Dir selbst über verschiedene Geräte teilst. Geht zB mit WhatsApp oder Telegram usw.
	- Vergewissere Dich, das Du die Datei dann später auch wieder auf Deinem Android Gerät auffinden wirst.
2.	Öffne die Cruiser App. Klicke auf das linke Symbol mit den "drei Balken".
	- Wähle "Meine Orte" und im Anschluss "Tracks" aus.
	- Vergewissere Dich, dass Du in dem 3 Punkt Ordner (siehe oben links, die 3 dicken Punkte) bist
	- Wenn Du die Tracks das erste Mal öffnest, wird hier bereits ein leerer Ordner mit Namen tracks angelegt sein.
	- Klicke nun zur Rechten auf die "3 Punkte" (des Track Ordners) und wähle "Neue Gruppe" aus. Tippe nun irgendeinen Namen ein (Ich empfehle Strassensperrungen) und wähle "Akzeptieren" (das Häkchen)
3.  Jetzt erscheint die "neue Gruppe". Klicke abermals auf die "3 Punkte" der neuen Gruppe.
	- Wähle nun "Importieren" aus. Danach musst Du natürlich wissen, wo sich Deine Datei befindet. Oft liegt sie unter downloads, oder wenn sie im Google Drive ist, findest Du
	  sie über die Cloud. Leider gibt es zu viele Möglichkeiten, die ich hier nicht aufzeigen kann.
	- Wenn Du die Datei gefunden hast, importiere sie. Wenn alles gut läuft und die Datei gut vorbereitet wurde, sind die Tracks nun in Deinem Gruppenordner verfügbar.
	- Im Vergleich zu der Track Ordnerstruktur unterscheidet sich die POI Ordnerstruktur etwas. In der POI Ordnerstruktur importierst Du eine Datei welche vermutlich viele POIs enthält. Im Cruiser bekommt der Ordner den Namen der POI gpx Datei, während in dem Ordner die ganzen POIs als Einzelpunkte ausgepackt werden (das können eine  Menge sein).  
       Bei Tracks ist es nun ähnlich aber doch etwas anders. Du legts einen neunen Ordner an und nennst ihn Straßensperrungen. In ihn importierts Du nun die Track gpx Datei. Jetzt siehst Du in dem Ordner (Straßensperrungen) tatsächlich auch nur diese eine GPX Datei. Das bedeutet, der Straßensperrungsordner fungiert als eine Art Überordner. Dort hinein kannst Du nun weitere Straßensperrungen (vielleicht aus anderen Ländern). Im Vergleich zu den POIs bleiben alle "Einzeltracks", welche sich in der gpx Track Datei befinden, darin bestehen und werden nicht wie bei einer POI gpx Datei als Einzelpunkte ausgepackt.
4. Um den Prozess abzuschließen, sollten wir noch ein paar Einstellungen ändern/vornehmen, damit wir das beste Ergebnis erzielen.
	- Aus diesem Grund wähle wieder die "3 Punke" der neuen Gruppe an und geben den Tracks noch eine eigene Farbe durch anwählen von "Farbe". Das macht sie auf der Karte noch besser unterscheidbar.
	- Zu guter Letzt das wohl wichtigste: Wir möchten die Tracks natürlich auch auf der Karte sehen. Dafür wählen wir "Auf der Karte anzeigen" an und aktivieren die 
	  Option "Auf der Karte anzeigen" aus. Zuletzt stellen wir noch das Zoom Level ein. Das ist die Ebene, ab der beim hineinzoomen die Tracks sichtbar werden. Von Haus aus ist das immer aus einem hohen Zoom heraus festgelegt. Bei vielen Tracks kann es dann aber passieren, dass man die Karte nicht mehr sieht. Aus diesem Grund empfehle ich hier ein Zoomlevel von 10 oder größer. Einfach mal selbst testen, wie es am besten passt.
	  
🙂 **Glückwunsch**. Du bist fertig. Ganz wichtig: Vergesse nicht zu Speichern indem DU unten rechts das Speichern Symbol anklickst, sonst ist wieder alles weg. Alles was oben beschrieben wurde, kann nachträglich auch wieder geändert werden. Wenn Du manche Tracks vorübergehend nicht anzeigen möchtest, brauchst Du sie nicht gleich zu löschen, deaktivieren sie einfach wieder, so wie Du sie zuletzt in der Karte bei der Navigation aktiviert hast. Das war's. Viel Spaß.

**Comment by DIrgi** [comment]
### Road closures for motor bikes (Straßensperrungen für Motorräder) (Part 1)
See part 2 a little bit further below (Teil 2 siehe weiter unten)

- (EN)
    - In Germany, Austria, Switzerland - shortly the DACH region (probably in some other countries as well) there are for several years temporary or seasonal road closures existing, only for motor bikes. It's not really funny when you drive a tour and then the road is blocked, only for you because you are a biker additionally, your way around is many kilometers and takes some time. Luckily when you are aware about and can drive the road at the right time or have the possibility to bypass. This is, what these POIs are for. They will display you on you map where you have to expect restrictions and what the requirements are. In the following example pictures you see how it can look like, depending which symbol and color you select (see installation instruction in the beginning of this tread). To receive more info about the closure and regulation, simply click on the Symbol.

- (DE)
    - In Deutschland, Österreich und der Schweiz - kurz DACH Region (evtl. auch in anderen Ländern) gibt es verschiedene temporäre Straßensperrungen nur für Motorradfahrer. Nichts ist ärgerlicher, wenn man auf einmal bei einer Tour vor solch einer Sperrung steht, nicht weiterkommt oder einen großen Umweg mit Zeitaufwand fahren muss. Glücklicher Weise wirst Du nun informiert sein, kannst diese Sperrungen umfahren oder Dich danach Richten, wann sie sind bzw. welche Auflagen es gibt. Symbole werde Dir in der Karte angezeigt und mit einem Klick darauf erfährst Du kurz und knapp die Auflagen. In den folgenden Beispielbildern siehst Du, wie es aussehen könnte. Das Symbol hierfür und die Farbe legst Du selber fest (nutze dazu die Installationsanleitung welche Eingangs in diesem Sammelthread steht). Um mehr Informationen zu den Sperrungen einer Strecke zu erfahren, klicke einfach auf das Symbol. Die relevanten Informationen werden Dir dann angezeigt.

<hr>
Some example pictures (ein paar Beispielbilder)

![image](https://github.com/devemux86/cruiser/assets/57968526/f3389dde-ec4d-4152-b461-b186dfccd9c8)
<br>
mouse click on symbol (Mausklick auf Symbol)
Number is zip code/ then the timing: Road is closed, each Saturday, Sunday / Place: from/to direction Feldberg
(Nummer ist PLZ / Dann die Zeit: Straße gesperrt am SA & SO / Ort: Straße von/nach Feldberg)
![image](https://github.com/devemux86/cruiser/assets/57968526/7f592861-4b32-48a1-9999-1623dac51cad)
<br>Closed the complete year for bikes louder then 95db(A) (Gesperrt für Motorräder, lauter als 95db(A))
![image](https://github.com/devemux86/cruiser/assets/57968526/812bd730-7961-466b-84f8-f1569febc556)

<hr>
Sperrungen zum herunterladen

[Streckensperrungen_Motorrad POI.zip](https://github.com/devemux86/cruiser/files/14932403/Streckensperrungen_Motorrad.POI.zip)

**Comment by DIrgi** [comment]
### Balkony roads in France (Balkonstraßen in Frankreich):

Do you like driving roads like these ?
(Magst Du es, solche Strecken zu fahren?)
![2021-08-FRA-Combe Laval 01](https://github.com/devemux86/cruiser/assets/57968526/d0230206-ee39-41e8-b031-704a79685aac)

![2021-08-FRA-Combe Laval 05](https://github.com/devemux86/cruiser/assets/57968526/6205cc6f-7977-4df2-872e-23817e28510b)

![2021-08-FRA-Combe Laval 07](https://github.com/devemux86/cruiser/assets/57968526/9a4f02a8-7b76-4581-b0cf-5dfb20a94698)

![2021-08-FRA-Gorges du Nan 06](https://github.com/devemux86/cruiser/assets/57968526/3f0fb268-1dfc-442f-928c-c1a0337b92b7)

Then my POIs will help you in France to find them 🙂
(Dann werden Dir meine POIs dabei helfen, diese in Frankreich zu finden) 🙂

![image](https://github.com/devemux86/cruiser/assets/57968526/2313c94a-2f69-4c81-ac9b-3fc6de68635c)


[French Balkony Roads.gpx.zip](https://github.com/devemux86/cruiser/files/14932535/French.Balkony.Roads.gpx.zip)

**Reply by bios4** [reply]
Stunning collection of POIs - thanks for sharing! 😀👍

**Comment by devemux86** [comment]
@DIrgi thanks for the impressive and detailed presentation! :+1:

**Reply by DIrgi** [reply]
Not finished yet, I have some more in pipeline. It's all about the time, as usual.

**Comment by DIrgi** [comment]
### Road closures for motor bikes (Straßensperrungen für Motorräder) (Part 2)
See part 1 a little bit further above (Teil 1 siehe oben)

- (EN)
The following tracks can be used as standalone but are ideal as a supplement to the already published POIs.
While the POIs show you where the regulations are with a sign on the map and offer you further information while clicking them,
Tracks mark you exactly the route which is in consideration. So at the end you can and should use both to have the best informative output on you map (sorry that I didn't add this already to the POIs, but I was first not aware if and how Cruiser can display this (thanks to Emux).
To install them in the right way, just follow the instructions for track installations I recently published them in the beginning of this document.
On the example map you see a screenshot how it will look like in combination with the POIs 
(see below for the gpx track file)

- (DE)
Die folgenden Tracks können alleinstehend benutzt werden, bieten aber den besten Mehrwert in Kombination mit den bereits erschienenen POIs (weiter oben). Während die POIs dir mit einem Symbol auf der Straße nur anzeigen, das es dort Einschränkungen gibt, Dir aber dazu auch noch weitere Informationen bieten, wenn Du sie anklickst, markieren Dir die Tracks auch den betroffenen Straßen Abschnitt (dafür haben sie meist keine weitern Informationen). Wenn Du am Ende nun beides benutzt: POIs und Tracks, hast Du die besten Informationen in Deiner Karte zur Verfügung die es gibt. (Sorry, dass ich das nicht bereits bei den POIs mit eingebaut hatte, aber mir fehlten schlichtweg die Informationen ob und wie Cruiser das umsetzen kann. (Dank Emux reiche ich das nun nach)
Um die Tracks zu installieren, folge der Anleitung (weiter oben) zum Thema Track Installation.
Im dem nun folgenden Bildausschnitt siehst Du beide POIs und Tracks in Aktion. So kann es dann auch bei Dir Aussehen.
(die gpx Track Datei findest Du am Ende)

![image](https://github.com/devemux86/cruiser/assets/57968526/c15eb20a-0184-48c6-82be-fade0e5b2a3d)

[Streckensperrungen_Motorrad Tracks DACH.gpx.zip](https://github.com/devemux86/cruiser/files/14938173/Streckensperrungen_Motorrad.Tracks.DACH.gpx.zip)

**Comment by Ciutnik** [comment]
It is worth reminding at this point about the wonderful tool [Overpass turbo](https://overpass-turbo.eu/).

It allows us to retrieve POIs from OpenStreetMap using queries. Thanks to **Wizard**, this trivially easy-to-use tool has great possibilities for fine-tuning the results, which we can export as GPX. Although the queries are formulated in "human" language (assuming basic knowledge of OSM tags), it is worth reading the manual to learn their syntax - this opens up unlimited possibilities for acquiring POIs.

A quick tip: if some of the results are polygons, and we are interested in points, changing the notation `out [geom / body / ...]` to `out center` in the last line of the query will generate points in the geometric center of the polygon. In the example below `toutism=viewpoint AND man_made=tower IN Subcarpatia`, I generated a collection of all the lookout towers in my beloved Subcarpatia region in 5 seconds :)

_Originally, the circles with yellow fill are POIs, and those with red fill were polygons (converted to points thanks to a trick)_

![overpass-turbo](https://github.com/devemux86/cruiser/assets/122647782/29692939-a995-4372-a6a8-16307f9518e5)

**Reply by DIrgi** [reply]
Great, I'll have a look on it 😀

**Reply by devemux86** [reply]
Another way is the overpass-turbo links at the right in the OpenStreetMap Wiki pages of the tags.
e.g. [fuel stations](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dfuel) have this [overpass-turbo](https://overpass-turbo.eu/?template=key-value&key=amenity&value=fuel)

**Reply by Ciutnik** [reply]
Well yes, I never noticed that link! It cleverly limits the area to the visible area of the window with the `{{bbox}}` parameter.

**Reply by r11d11** [reply]
@Ciutnik Great tip with out center. I had always problems to import the polygons, but now it is easy. Thank you.
